src/AppBundle/Resources/views/Ndr/DeputyExpense/_list.html.twig

```diff
@@ -38,44 +40,47 @@
             </a>
         </div>
     </div>
-{% endif %}
 
-{% if ndr.expenses | length > 0 %}
-    <table class="check-your-answers columns-3">
-        <thead>
-            <tr>
-                <th>Description of expense</th>
-                <th class="text--right">Amount</th>
-                <th>
-                    <span class="visually-hidden">{{ 'actions' | trans({}, 'common') }}</span>
-                </th>
-            </tr>
-        </thead>
-        <tbody>
-            {% for expense in ndr.expenses %}
-                <tr class="behat-region-expense-{{ expense.explanation | behat_namify }}">
-                    <td class="">
-                        {{ expense.explanation }}<br/>
-                    </td>
-                    <td class="numeric-small">
-                        £{{ expense.amount | money_format }}
-                    </td>
-                    <td class="change-answer width-sixth soft-half--left hard--right">
-                        <a href="{{ url('ndr_deputy_expenses_edit', { 'ndrId': ndr.id, 'expenseId': expense.id }) }}" class="behat-link-edit action-link">{{ 'edit' | trans({}, 'common') }}</a>
-                        <a href="{{ path('ndr_deputy_expenses_delete', { 'ndrId': ndr.id, 'expenseId': expense.id }) }}" class="push-half--left action-link behat-link-delete">{{ 'remove' | trans({}, 'common' ) }}</a>
-                    </td>
-                </tr>
-            {% endfor %}
-            <tr class="no-border">
-                <td>
-                    <span class="bold-small">Total amount</span>
-                </td>
-                <td class="numeric-small">
-                    <span class="bold-small behat-region-expense-total">£{{ ndr.expensesTotalValue | money_format }}</span>
-                </td>
-                <td></td>
-            </tr>
-        </tbody>
-    </table>
+    <dl class="govuk-summary-list columns-2 push-double--bottom">
```

Comment:
Looks like this should be wrapped in `{% if ndr.expenses | length > 0 %}` as before change